sstables/sstable_directory.cc

```diff
@@ -676,7 +672,7 @@ future<sstable_directory::pending_delete_result> sstable_directory::create_pendi
             base_dir.sync(general_disk_error_handler).get();
             dirlog.debug("{} written successfully.", pending_delete_log);
 
-      return res;
+        return pending_delete_log;
```

Comment:
Indentation seems off.

Reply:
In fact, the indentation was off for the whole function even _before_ this patch. This particular change is  "one-line indentation fix while at it". Here's how it looks like with this PR applied (I highlighted the changed line on the screenshot)

![image](https://github.com/user-attachments/assets/62eaa02e-63d3-4d1e-a586-ec76a718afc2)



The above several lines also have broken indentation, but I didn't fix it here not to pollute the patch.